buf/registry/module/v1/label_service.proto

```diff
@@ -143,6 +143,11 @@ message ListLabelHistoryRequest {
   uint32 page_size = 1 [(buf.validate.field).uint32.lte = 250];
   // The page to start from.
   //
+  // The value may be a page_token returned from ListLabelHistoryResponse,
+  // or it may be a Commit id.
+  //
+  // If a Commit id, the returned history will start at that Commit.
```

Comment:
Nit: Add a comment saying that if the passed Commit id is not part of the label history, this returns an error.